File: 프로그래머스/lv0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    if price >= 100000 and price < 300000:
        return int(price * 0.95)
    if price >= 300000 and price < 500000:
        return int(price * 0.90)
    if price >= 500000:
        return int(price * 0.80)
    else :
        return price

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.swift
import Foundation

func solution(_ price:Int) -> Int {
    var rst = Double(price)
    var sale = 0.0
    
    
    // over 100_000
    if price >= 100_000 {
        sale = rst * 0.05
    }
    
    // over 300_000
    if price >= 300_000 {
        sale = rst * 0.1
    } 
    
    // over 500_000
    if price >= 500_000 {
        sale = rst * 0.2
    } 
    
    return Int(rst - sale)
}